frontend/tailwind.config.ts

```diff
@@ -101,6 +101,7 @@ export default {
       'time-purple': 'rgb(var(--time-purple) / <alpha-value>)',
       'time-yellow': 'rgb(var(--time-yellow) / <alpha-value>)',
       'toggle-color': 'rgb(var(--toggle-color) / <alpha-value>)',
+      'transparent': '#00000000',
```

Comment:
the Tailwind custom themes were overwriting instead of extending. I have moved `colors` and `borderRadius` objects into the `extend` property. That way devs have access to built-ins - however i can revert this if you did not want these available.

Reply:
ah gotcha; i didnt think about that overwriting the built in transparent; good call. either way is fine for now; eventually down the line we'll want to switch everything over from `extend` to overwriting, simply to lock in the design system and not have someone accidentally use the tailwind green instead of the het alt-green. but for now whichever way you prefer works for me. thanks!